### PR TITLE
Update to openshift/api/v1 deps

### PR DESCRIPTION
**Pull request by coreydaley**

### CODE REVIEW

File: config/config.yaml [hunk 1]
```diff
@@ -1,3 +1,3 @@
-apiVersion: imageregistry.operator.openshift.io/v1alpha1
+apiVersion: imageregistry.operator.openshift.io/v1
```
Comment (bparees) [inline config/config.yaml]: why wasn't this part of your other PR?  (and how did your other PR pass tests w/o this, or did it?)

Comment (bparees) [reply config/config.yaml]: nm, i see this is from the other commit you pulled in.

**Comment by coreydaley** [comment]
@bparees ptal
I split out the vendor bump and the code changes into two commits

File: pkg/apis/imageregistry/v1/types.go [hunk 2]
```diff
@@ -90,7 +90,7 @@ type ImageRegistryConfigRoute struct {
 }
 
 type ImageRegistrySpec struct {
-	operatorsv1alpha1api.OperatorSpec `json:",inline"`
+	operatorsv1api.OperatorSpec `json:",inline"`
```
Comment (bparees) [inline pkg/apis/imageregistry/v1/types.go]: what are we still using from this? can we get rid of it entirely?

Comment (coreydaley) [reply pkg/apis/imageregistry/v1/types.go]: Checking...

**Comment by bparees** [comment]
squash the last 2 commits and lgtm.

**Comment by coreydaley** [comment]
@bparees squashed and pushed, thanks!

**Pull request merged**